storybook/src/pages/amsterdam/ArticlePage/ArticleBody.tsx

```diff
@@ -94,7 +94,7 @@ export const ArticleBody = ({ lead, paragraph1, spotlightHeading, spotlightLinkL
             </Paragraph>
           </section>
           <section>
-            <Heading className="ams-mb-xs" level={2}>
+            <Heading className="ams-mb-m" level={2}>
```

Comment:
Let‘s make the space between Headings and Lists small as well. Paragraph, * List and Standalone Link should all behave the same.

```suggestion
            <Heading className="ams-mb-s" level={2}>
```

Reply:
I you change the spreadsheet I can process the changes.

Reply:
Done.